delira/data_loading/sampler/batch.py

```diff
@@ -0,0 +1,65 @@
+from delira.data_loading.sampler.abstract import AbstractSampler
+
+
+class BatchSampler(object):
+    """
+    A Sampler-Wrapper combining the single indices sampled by a sampler to
+    batches of a given size
+    """
+
+    def __init__(self, sampler: AbstractSampler, batch_size, truncate=False):
+        """
+
+        Parameters
+        ----------
+        sampler : :class:`AbstractSampler`
+            the actual sampler producing single-sized samples
+        batch_size : int
+            the size of each batch
+        truncate : bool
```

Comment:
Nit: use consistent naming of this argument (in `DataManager` it is called `drop_last ` which i prefer)